README.md

```diff
@@ -1 +1,112 @@
-# hello-microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" /> -->
+  <br/>
+
+  <h3><b>Hello-Microverse Project </b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**Hello Microverse** is a project built to learn how to configure html&CSS linters
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack">HTML & CSS</a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjXwZ2LqJX9AhXoUqQEHWcNBS8QFnoECAoQAQ&url=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FWeb%2FHTML&usg=AOvVaw23zU8Os1zIEwrE-SW_k5x4">HTML</a></li>
+    <li><a href="https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiEo8eZqJX9AhWkTKQEHRF0CwUQFnoECAkQAQ&url=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FWeb%2FCSS&usg=AOvVaw2uGFFuZmjPce8PuYb0p8Tp">CSS</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/rosemutai)
+- Twitter: [@twitterhandle](https://twitter.com/__rosemutai__)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/rosemutai)
+
```

Comment:
- [ ] As mentioned by the previous reviewer, please add all sections provided in the README Template. Look at each item in the table of content, and ensure that each section is present in your README and well-edited. It should be arranged in the order it was arranged in the Template. The following sections are missing:
 - [Key Features](#key-features)
  - [🚀 Live Demo](#live-demo)
- [💻 Getting Started](#getting-started)
  - [Setup](#setup)
  - [Prerequisites](#prerequisites)
  - [Install](#install)
  - [Usage](#usage) 
- [👥 Authors](#authors)
- [🔭 Future Features](#future-features)
- [🤝 Contributing](#contributing)
- [⭐️ Show your support](#support)
- [🙏 Acknowledgements](#acknowledgements)
- [❓ FAQ (OPTIONAL)](#faq)
- [📝 License](#license)

_**Also, note that all section in the README is compulsory except the FAQ section. Kindly edit the template accordingly.**_

- [ ] Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it. 🙏
- [ ] Please add your project title.
- [ ] For the Tech Stack Section, Kindly remove the Server and Database subsections because the technologies listed there are not relevant to this project. For the Client section, kindly include HTML and CSS 👍
- [ ] For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. 👍
- [ ] For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:

    -  Installation of Git
    -  Installation of NPM
    -  A working browser
    -  Internet connection
    -  A code editor like Visual Studio Code
    -  Installation of Live Server plugging in Visual Studio Code
    -  Any other tool you can think about

- [ ] For the Setup section, kindly add instructions for cloning your repository on a local machine 👍
- [ ] For the Install section, kindly add instructions for setting up linters used in this project
- [ ] For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.
- [ ] For the **_Author_** section, please edit it with your GitHub and social media handles.
- [ ] For the MIT section, kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE - https://choosealicense.com/licenses/mit/

**_Kindly implement all the things listed above. It will make your README.md file look professional and it will give you an idea of how to set README.md files for most of the projects you will execute as a Microverse student._** 👍 💻